tf/do/main.tf

```diff
@@ -0,0 +1,220 @@
+provider "digitalocean" {
+    token = var.do_token
+}
+
+resource "random_id" "droplet_id" {
+    byte_length = 8
```

Comment:
I actually want to name the droplet based on uptimerobot rule to make it easier to identify.